episodes/chapter4.md

```diff
@@ -202,45 +143,25 @@ You will see a message:
 
 ![Collaboration details](fig/create_collab_03.png)
 
-### Reset API key for a node
-If you want to reset the API key for a node, you can do so by following these steps:
-
-1. Click on the `Nodes` tab in the administration page.
-2. Click on the tab of the node you want to reset the API key for in the list of nodes.
-
-![Reset API key for a node](fig/reset_api_key_01.png)
-
-3. Click on the `Reset API key` button.
-    * You will see a dialog box to ask you to download the new API key.
-
-![Download new API key](fig/reset_api_key_02.png)
-
-You will see a message:
-
-> API key download
->
-> Your API key has been reset. Please read your new key in the file that has been downloaded.
-
-At this point, put the new API key in your node configuration file and restart the node to connect to the server again.
 
 ::::::::::::::::::::::::::::::::::::: challenge
 
-## Challenge 3: Manage vantage6 project using the UI
+## Challenge 2: Manage vantage6 project using the UI
 
-Go the administration page in the vantage6 UI and try to create a new organization, a new user, and a new collaboration. Also, try to reset the API key for a node.
+Go to the administration page in the vantage6 UI, and try to do the following:
```

Comment:
nitpicking: maybe delete the words `try to`
That way we show that we have faith in our participants :-)